microservices/first/main.py

```diff
@@ -0,0 +1,78 @@
+from flask import Flask, request, jsonify
+from db_manager import DBManager
```

Comment:
To keep this consistent with the other library call, I would delete this line and use the database module through the `utils` library load
```suggestion
```